internal/commands/app/create.go

```diff
@@ -22,12 +23,47 @@ type CommandCreate struct {
 func (cmd *CommandCreate) Flags(fs *pflag.FlagSet) {
 	fs.StringVar(&cmd.inputs.Project, flagProject, "", flagProjectUsage)
 	fs.StringVarP(&cmd.inputs.Name, flagName, flagNameShort, "", flagNameUsage)
+	fs.StringVarP(&cmd.inputs.From, flagFrom, flagFromShort, "", flagFromUsage)
 	fs.StringVarP(&cmd.inputs.Directory, flagDirectory, flagDirectoryShort, "", flagDirectoryUsage)
 	fs.VarP(&cmd.inputs.DeploymentModel, flagDeploymentModel, flagDeploymentModelShort, flagDeploymentModelUsage)
 	fs.VarP(&cmd.inputs.Location, flagLocation, flagLocationShort, flagLocationUsage)
 	fs.StringVarP(&cmd.inputs.DataSource, flagDataSource, flagDataSourceShort, "", flagDataSourceUsage)
-	// TODO(REALMC-8134): Implement dry-run for app create command
-	// fs.BoolVarP(&cmd.inputs.DryRun, flagDryRun, flagDryRunShort, false, flagDryRunUsage)
+	fs.BoolVarP(&cmd.inputs.DryRun, flagDryRun, flagDryRunShort, false, flagDryRunUsage)
+}
+
+func (cmd *CommandCreate) commandString(omitDryRun bool) string {
+	sb := strings.Builder{}
+	sb.WriteString("app create")
```

Comment:
should probably prefix this with `realm-cli` ... and thinking on it, we probably want that to be shared with the `cliName` const we currently have in `cmd/root.go`.

let's move that `const` to the `internal/cli` package and make it a `var` instead.  when we ultimately build our CLI, we can use build-time substitution to set this `var` to our actual CLI name (will still be `realm-cli` at this point), and then our commands here (this isn't the first "command string" we've tried to build) can reference `cli.Name` to get that same name value.

that way, we're consistent with our names no matter what it ends up being

---

also, meganit, but can we move this function towards the bottom of the file, at least after the `Handler` code?